bigtable/google/cloud/bigtable/table.py

```diff
@@ -45,6 +49,22 @@ class TooManyMutationsError(ValueError):
     """The number of mutations for bulk request is too big."""
 
 
+class _MutateRowsRetryableError(Exception):
+    """A retryable error in Mutate Rows response."""
+
+    def __init__(self, retryable_responses=None):
+        super(_MutateRowsRetryableError, self).__init__()
+        self.retryable_responses = retryable_responses or []
+
+
+MUTATE_ROWS_DEFAULT_RETRY = Retry(
```

Comment:
I guess we wanna place the constants at the top of the file as a convention, so this is not ideal 'cos it needs to have seen `_MutateRowsRetryableError`. I welcome suggestion for this please.

I'm also thinking if it's a good idea to rename `_MutateRowsRetryableError` into just a generic `_RetryableError` and remove the `retryable_responses` (since it's not used by the `google.core.api.Retry` anyway.

Reply:
> I guess we wanna place the constants at the top of the file as a convention, so this is not ideal 'cos it needs to have seen _MutateRowsRetryableError. I welcome suggestion for this please.

As long as the linter is happy it's fine.

> I'm also thinking if it's a good idea to rename _MutateRowsRetryableError into just a generic _RetryableError and remove the retryable_responses (since it's not used by the google.core.api.Retry anyway.

If you do that then I would just use one of the many existing exception types in `google.api.core.exceptions`.